activerecord/lib/active_record/sanitization.rb

```diff
@@ -36,8 +36,7 @@ def sanitize_sql_for_conditions(condition)
           else        condition
           end
         end
-        alias_method :sanitize_sql, :sanitize_sql_for_conditions
-        alias_method :sanitize_conditions, :sanitize_sql
```

Comment:
While it is only used internally in one place it is a public API method. Could you add a deprecation message and a CHANGELOG entry?